0.data-download/scripts/data_loader.py

```diff
@@ -10,22 +10,22 @@
 def load_data(data_directory, adult_or_pediatric = "all"):
 
     # Define data paths
-    sample_info_file = pathlib.Path(data_directory, "age_visual_sample_info.csv")
+    data_directory = "../0.data-download/data/"
+    sample_info_file = pathlib.Path(data_directory, "sample_info_age_column_cleaned.csv")
     dependency_data_file = pathlib.Path(data_directory, "CRISPR_gene_dependency.csv")
 
     # Load data
     dependency_df = pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(1)
     sample_df = pd.read_csv(sample_info_file, index_col=0)
-    sample_df = sample_df.set_index("DepMap_ID").reset_index()
+
     
     # rearrange sample info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
-    sample_df_sort = sample_df.set_index("DepMap_ID")
-    sample_df_sort = sample_df_sort.sort_index(ascending=True)
-    sample_df = sample_df_sort.reset_index()
+    sample_df = sample_df.sort_index(ascending=True)
+    sample_df = sample_df.reset_index()
     
-    dependency_df_sort = dependency_df.set_index("DepMap_ID")
-    dependency_df_sort = dependency_df_sort.sort_index(ascending=True)
-    dependency_df = dependency_df_sort.reset_index()
+    dependency_df = dependency_df.set_index("DepMap_ID")
+    dependency_df = dependency_df.sort_index(ascending=True)
+    dependency_df = dependency_df.reset_index()
```

Comment:
```suggestion
    dependency_df = (
        dependency_df
        .set_index("DepMap_ID")
        .sort_index(ascending=True)
        .reset_index()
    )
```

Here is a quick lesson in pandas chaining, which I love

Reply:
please test this, and also apply [`black`](https://github.com/psf/black)